packages/web-components/src/components/ic-data-table/ic-data-table.tsx

```diff
@@ -90,6 +101,15 @@ export class DataTable {
    */
   @Prop() embedded?: boolean = false;
 
+  /**
+   * Allows for custom setting of row heights on individual rows based on an individual value from the `data` prop and the row index.
+   * If the function returns `null`, that row's height will be set to the `rowHeight` property.
+   */
+  @Prop({ mutable: true }) getRowHeight?: (parmas: {
```

Comment:
Feels weird to me that a function called 'get' would be used for setting a value. Can you explain the logic here?

Reply:
I think the logic is that calling the function returns (gets) the number to be used for the row height. The actual logic for setting it is done a bit further down (line 587)

Reply:
Thanks for explaining. I'm still a bit confused since this is a mutable property (rather than a function, which I assume all "getX" methods would be) - I guess my next question is: could this prop be called just "rowHeight"?  

I've done a global search of the project in VSCode and there's no other prop that starts with "get"

Reply:
rowHeight is already used by another property which sets the height for all rows. The fact that it is mutable is because this prop is overridden on line 309. I think rowHeight would be misleading for this as its about getting the height for rows that meet a specific criteria, rather than setting it more generally, if that makes sense

Reply:
Then i'd suggest 'rowHeight' should be renamed to 'universalRowHeight' or 'allRowHeights' 
and 'getRowHeight' to 'rowHeight' to better differentiate the scopes between them.

Sorry for not understanding the full picture before now!

Reply:
Have updated to globalRowHeight and variableRowHeight